services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
```

Comment:
no need to make queries to db here, use lookups instead

Reply:
Done